src/Package/ReferenceAssemblies/Microsoft.Build/netstandard/Microsoft.Build.cs

```diff
@@ -1057,7 +1057,7 @@ public enum NodeEngineShutdownReason
     }
     public partial class OutOfProcNode
     {
-        public OutOfProcNode(string clientToServerPipeHandle, string serverToClientPipeHandle) { }
+        public OutOfProcNode() { }
```

Comment:
Uh. I guess this is a breaking change? No one should have been using this class. Should we add an overload that just ignores the input strings for theoretical compat?

Reply:
yeah probably, although I can't find anyone who is using this class.